public/css/main.css

```diff
@@ -159,41 +161,41 @@ h1.header a{
   white-space: pre-line;
 }
 
-.topic .question{
-  margin-top: 20px;
+.topic .question {
+  margin-top: 48px;
```

Comment:
This line is the change we are going for. Add some spacing in between the questions.